crates/fuel-core/src/graphql_api/database.rs

```diff
@@ -88,8 +88,10 @@ pub struct ReadDatabase {
     on_chain: Box<dyn AtomicView<LatestView = OnChainView>>,
     /// The off-chain database view provider.
     off_chain: Box<dyn AtomicView<LatestView = OffChainView>>,
-    /// The flag that indicates whether the Balances cache table is enabled.
-    balances_enabled: bool,
+    /// The flag that indicates whether the Balances indexation is enabled.
+    balances_indexation_enabled: bool,
```

Comment:
My usual approach is _"generalize if you need more than 2"_. If we ever gonna add one more I will probably try to re-use the approach taken at the DB metadata level ([here](https://github.com/FuelLabs/fuel-core/pull/2463/files#diff-33f0910104b2e2e4219d73505f68c40b804e1bc540c9d8e3d774a8cd73ab82dcR83-R85)).

I'm not super strong at sticking to my approach, though, so happy to hear other opinions :)

Also, for this case, since we don't need to care that much about the storage, I'd probably not go with bitmask, but a regular `HashSet`.